pkg/cache/fs_test.go

```diff
@@ -100,11 +101,11 @@ func TestFSCache_GetBlob(t *testing.T) {
 			fs, err := NewFSCache(tmpDir)
 			require.NoError(t, err)
 			defer func() {
-				_ = fs.Clear()
+				_ = fs.Clear(context.Background())
```

Comment:
nit:
```suggestion
				_ = fs.Clear(t.Background())
```

and same for other tests